src/transform/regress_out/script.py

```diff
@@ -43,12 +43,21 @@ def setup_logger():
     mod = par["modality"]
     logger.info("Regress out variables on modality %s", mod)
     data = mdata.mod[mod]
+    sc_data = data.copy()
```

Comment:
Can you create this adata from scratch (with an X and the obs) as not to copy unnecessary data? :bow:

Reply:
Oic! :+1: